salt/cloud/cli.py

```diff
@@ -402,13 +402,6 @@ def run(self):
                     self.config['pki_dir'], vm_['pub_key'], key_id
                 )
 
-            if 'os' not in vm_:
```

Comment:
Oh, you're right. I was thinking we needed that block here for the drivers that don't call `salt.utils.cloud.bootstrap` but that should be covered in the `create` function in `salt/cloud/__init__.py`